tests/test_simplify.py

```diff
@@ -0,0 +1,149 @@
+"""Test for petab.simplify.*"""
+from math import nan
+
+import pandas as pd
+import pytest
+import simplesbml
+from pandas.testing import *
+
+from petab import Problem
+from petab.C import *  # noqa: F403
+from petab.models.sbml_model import SbmlModel
+from petab.simplify import *
+
+
+@pytest.fixture
+def problem() -> Problem:
+    ss_model = simplesbml.SbmlModel()
+    ss_model.addParameter("some_parameter", val=1.0)
+    ss_model.addParameter("same_value_for_all_conditions", val=1.0)
+
+    observable_df = pd.DataFrame(
+        {
+            OBSERVABLE_ID: ["obs_used", "obs_unused", "obs_used_2"],
+            OBSERVABLE_FORMULA: [1.0, 2.0, 3.0],
+            NOISE_FORMULA: [1.0, 2.0, 3.0],
+        }
+    )
+    observable_df.set_index(OBSERVABLE_ID, inplace=True)
```

Comment:
Could use `get_observable_df`

Reply:
I found it more explicit this way. Undecided.